### PR TITLE
added 4 free books (courses?) in polish

### PR DESCRIPTION
**Pull request by kri99**
## Hacktoberfest notes:

- due to volume of submissions, we may not be able to review PRs that do not pass tests and do not have informative titles.
- please read our [contributing guidelines](/CONTRIBUTING.md)
- be sure to check the output of Travis-CI for linter errors
- if this is your first open source contribution, make sure it's not your last!

## What does this PR do?
Add Resource(s) | Remove Resource(s) | Add info | Improve Repo

### Checklist:
- [x] Not a duplicate
- [x] Included author(s) if appropriate
- [x] Lists are in alphabetical order
- [x] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

File: free-programming-books-pl.md [hunk 1]
```diff
@@ -38,6 +38,7 @@
 
 ### Android
 
+* [O Androidzie ludzkim głosem](http://andrzejklusiewicz-android.blogspot.com/p/bezpatny-kurs-programowania-android-java.html) - Andrzej Klusiewicz
```
Comment (SethiPandi) [inline free-programming-books-pl.md:41]: It looks like this source is accessible via HTTPS, could we please use the https:// link instead?

```suggestion
* [O Androidzie ludzkim głosem](https://andrzejklusiewicz-android.blogspot.com/p/bezpatny-kurs-programowania-android-java.html) - Andrzej Klusiewicz
```

File: free-programming-books-pl.md [hunk 2]
```diff
@@ -81,6 +83,7 @@
 ### CSS
 
 * [Kaskadowe Arkusze Stylów](http://www.kurshtml.edu.pl/css/index.html) - Sławomir Kokłowski
+* [Kurs CSS](http://webref.pl/arena/css/css_index.html) - Arkadiusz Michalski
```
Comment (SethiPandi) [inline free-programming-books-pl.md:86]:
```suggestion
* [Kurs CSS](https://webref.pl/arena/css/css_index.html) - Arkadiusz Michalski
```

File: free-programming-books-pl.md [hunk 3]
```diff
@@ -128,6 +131,7 @@
 ### Perl
 
 * [Perl](https://pl.wikibooks.org/wiki/Perl) - Wikibooks
+* [Samouczek Perl](http://www.w3big.com/pl/perl/default.html)
```
Comment (SethiPandi) [inline free-programming-books-pl.md:134]:
```suggestion
* [Samouczek Perl](https://www.w3big.com/pl/perl/default.html)
```

**Review by SethiPandi — CHANGES_REQUESTED** [review]
Just left a few comments for you!

In general, we prefer HTTPS links if that's available for the webpage. Could the links that are HTTPS accessible please be updated?

**Comment by eshellman** [comment]
3 of these are labeled as courses. Maybe they should go on the courses-pl page?

**Review by davorpa — APPROVED** [review]
Review of stale PR with conflicts moved to #5513

Conflicts due to repo reorganization made on Dec 2020 (#5199).

**Pull request closed without merging**